docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -179,236 +179,463 @@ This will return a large amount of variable data, which may look like this, as t
-        "ansible_all_ipv4_addresses": [
-            "REDACTED IP ADDRESS"
-        ],
-        "ansible_all_ipv6_addresses": [
-            "REDACTED IPV6 ADDRESS"
-        ],
-        "ansible_architecture": "x86_64",
-        "ansible_bios_date": "09/20/2012",
-        "ansible_bios_version": "6.00",
-        "ansible_cmdline": {
-            "BOOT_IMAGE": "/boot/vmlinuz-3.5.0-23-generic",
-            "quiet": true,
-            "ro": true,
-            "root": "UUID=4195bff4-e157-4e41-8701-e93f0aec9e22",
-            "splash": true
-        },
-        "ansible_date_time": {
-            "date": "2013-10-02",
-            "day": "02",
-            "epoch": "1380756810",
-            "hour": "19",
-            "iso8601": "2013-10-02T23:33:30Z",
-            "iso8601_micro": "2013-10-02T23:33:30.036070Z",
-            "minute": "33",
-            "month": "10",
-            "second": "30",
-            "time": "19:33:30",
-            "tz": "EDT",
-            "year": "2013"
-        },
-        "ansible_default_ipv4": {
-            "address": "REDACTED",
-            "alias": "eth0",
-            "gateway": "REDACTED",
-            "interface": "eth0",
-            "macaddress": "REDACTED",
-            "mtu": 1500,
-            "netmask": "255.255.255.0",
-            "network": "REDACTED",
-            "type": "ether"
+NEW:
+
+{
+    "ansible_all_ipv4_addresses": [
+        "REDACTED IP ADDRESS"
+    ],
+    "ansible_all_ipv6_addresses": [
+        "REDACTED IP ADDRESS"
+    ],
+    "ansible_apparmor": {
+        "status": "disabled"
+    },
+    "ansible_architecture": "x86_64",
+    "ansible_bios_date": "11/28/2013",
+    "ansible_bios_version": "4.1.5",
+    "ansible_cmdline": {
+        "BOOT_IMAGE": "/boot/vmlinuz-3.10.0-862.14.4.el7.x86_64",
+        "console": "ttyS0,115200",
+        "no_timer_check": true,
+        "nofb": true,
+        "nomodeset": true,
+        "ro": true,
+        "root": "LABEL=cloudimg-rootfs",
+        "vga": "normal"
+    },
+    "ansible_date_time": {
+        "date": "2018-10-25",
+        "day": "25",
+        "epoch": "1540469324",
+        "hour": "12",
+        "iso8601": "2018-10-25T12:08:44Z",
+        "iso8601_basic": "20181025T120844109754",
+        "iso8601_basic_short": "20181025T120844",
+        "iso8601_micro": "2018-10-25T12:08:44.109968Z",
+        "minute": "08",
+        "month": "10",
+        "second": "44",
+        "time": "12:08:44",
+        "tz": "UTC",
+        "tz_offset": "+0000",
+        "weekday": "Thursday",
+        "weekday_number": "4",
+        "weeknumber": "43",
+        "year": "2018"
+    },
+    "ansible_default_ipv4": {
+        "address": "REDACTED",
+        "alias": "eth0",
+        "broadcast": "REDACTED",
+        "gateway": "REDACTED",
+        "interface": "eth0",
+        "macaddress": "REDACTED",
+        "mtu": 1500,
+        "netmask": "255.255.255.0",
+        "network": "REDACTED",
+        "type": "ether"
+    },
+    "ansible_default_ipv6": {},
```

Comment:
Possibly a nit-pick, but it could help to have the default ipv6 fields shown here as well, for completeness' sake.